### PR TITLE
Remove Person model and update tests

### LINKED ISSUE #83: Remove Person model
The Person model is no longer required for our application

### PR DESCRIPTION
**Pull request by jloh02**
Fix #83

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddCommand.java
Comment (clarissatjx) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:1]: we not doing the AddCommand<T> thing is it (similar to FindCommand<T>)

File: src/test/data/JsonPawPatrolStorageTest/invalidOwnerPawPatrol.json
Comment (clarissatjx) [inline src/test/data/JsonPawPatrolStorageTest/invalidOwnerPawPatrol.json:1]: thanks for updating all the test cases and json files also :')

**Review by clarissatjx — APPROVED** [review]
lgtm, looks good to merge

File: src/main/java/seedu/address/logic/commands/AddCommand.java
Comment (jloh02) [reply src/main/java/seedu/address/logic/commands/AddCommand.java:1]: Yes, refactoring can be done in the later weeks but the original AddCommand was simply to add Person

**Pull request merged**